client/client.go

```diff
@@ -0,0 +1,37 @@
+package main
```

Comment:
it looks like this is the http client, but i dont think this is one of the requirements for building up an echo server. you could just use something like curl or postman instead. we won't be building an http client on the http server work either. if you want something to test against your server you could write up a bash script with a bunch of curl requests

Reply:
Wasn't able to get this working w/ curl or postman, as this was the response to `curl localhost:8000 -i "hello"`:

curl: (1) Received HTTP/0.9 when not allowed

HTTP/1.1 200 OK
X-Correlation-Id: aaaaaaaa
Date: Wed, 01 Dec 2021 17:25:03 GMT
Content-Length: 0
Connection: close

Reply:
I think its worth looking more into this. if we are building the http server on top of this you will need it to handle curl/postman.

Reply:
understood